integration_tests/src/test/kotlin/integration/AllTestFilteredIT.kt

```diff
@@ -29,6 +30,7 @@ class AllTestFilteredIT {
     }
 
     @Test
+    @Ignore("Should be fixed in https://github.com/Flank/flank/issues/1388")
```

Comment:
should we ignore this test for all OS ?

Reply:
only for ios, android works but output could change after #1388

Reply:
I wonder if this will be picked up and removed in that ticket. Can we add a // TODO with a ticket number